# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dvbstreamer/dvbstreamer-2.1.0-libev-check.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- dvbstreamer-2.1.0/configure.in.orig	2011-05-06 10:55:04.640001222 +0200
+++ dvbstreamer-2.1.0/configure.in	2011-05-06 12:42:13.123997973 +0200
@@ -188,7 +188,19 @@
 dnl Check for libev
 dnl ---------------------------------------------------------------------------
 AC_CHECK_HEADER([ev.h], [], [AC_MSG_FAILURE([libev header file not found])])
-AC_CHECK_LIB([ev], [ev_loop], [], [AC_MSG_FAILURE([libev not found])])
+AC_CHECK_DECLS([ev_loop,
+		ev_loop_destroy,
+		ev_unloop,
+		ev_timer_init,
+		ev_timer_start,
+		ev_timer_stop,
+		ev_io_init,
+		ev_io_start,
+		ev_io_stop,
+		ev_async_init,
+		ev_async_start,
+		ev_async_stop,
+		ev_async_send], [], [AC_MSG_FAILURE([libev not usable])], [[#include <ev.h>]])
 LIBS=""
 
 dnl ---------------------------------------------------------------------------
